content/docs/04.5-devx/04-resource-quota.md

```diff
@@ -34,14 +33,14 @@ The developer quota can be managed through the Palette Platform as below:
 
 3. Set the desired *User Quotas* values.  The user quotas have the following default values:
 
-|**Resource Requirement for 2 Sandbox Clusters**|**Default Quota**|**Minimum Value**|
+|**Resource Requirement for two PVC**|**Default Quota**|**Minimum Value**|
```

Comment:
PVC is a common term in K8s domain which is for "persistent Volume Claim" , so may be we should not be abbreviating 'Palette Virtual Clusters' as PVC.

<img width="620" alt="image" src="https://user-images.githubusercontent.com/49595451/200457334-f24139fb-d85d-47ec-9e07-1db740837b56.png">